pkg/asset/templates/content/bootkube/pull.go

```diff
@@ -62,3 +63,58 @@ func (t *Pull) Load(f asset.FileFetcher) (bool, error) {
 	t.FileList = []*asset.File{file}
 	return true, nil
 }
+
+const (
+	openshiftConfigSecretPullSecretFileName = "openshift-config-secret-pull-secret.yaml.template"
+)
+
+var _ asset.WritableAsset = (*OpenshiftConfigSecretPullSecret)(nil)
+
+// OpenshiftConfigSecretPullSecret is the constant to represent contents of openshift-config-secret-pull-secret.yaml.template file.
+type OpenshiftConfigSecretPullSecret struct {
```

Comment:
Even though we don't use the manifest-templates stage any more, this type should be added to the targets for that stage.